.github/workflows/c-tests.yml

```diff
@@ -50,6 +50,11 @@ jobs:
       - name: Install RTI
         uses: ./.github/actions/install-rti
         if: ${{ runner.os == 'macOS' || runner.os == 'Linux' }}
+      - name: Build RTI docker image;
```

Comment:
I think it would be better to put this action in `./github/actions/install-rti` (or to create a separate action called `./github/actions/rti-docker`).